meetings/2024-04-10.md

```diff
@@ -0,0 +1,127 @@
+URL: https://github.com/orgs/solid/projects/16/views/1
+
+
+### WG Charter Update
+
+* hz: none today
+
+
+### WIP Implementation Feedback
+
+### Discussion #555
+URL: https://github.com/solid/specification/issues/635
+
+* HZ: Virginia marked #555 as `archived`, I think this issue can be closed
+* HZ: no objections, will close
+* RG: what about 635? let's ask Ted what he needs done
+* HZ: let's wait for Ted
+
+### WG Charter
+
+#### Tantek's feedback re 2 / 3 chairs
+URL: https://github.com/solid/solid-wg-charter/issues/66
+
+* VB: PAC, any status update on this?
+* HZ: I don't think is up to them (AB?) to decide how the W3C team selects it's chairs
+* HZ: I don't think it's also up to the CG, we don't need to deal with it
+* RG: PAC is not here
+* eP: let's better wait for PAC
+
+
+#### Add Explainer
+URL: https://github.com/solid/solid-wg-charter/issues/52
+
+* VB: Can this issue be closed given reference to *an explainer* in https://github.com/solid/solid-wg-charter/blob/004ba63e1b76715a4c72b5d6147c9eca6a9c7ee5/charter/index.html ?
+* RG: I asked Tim to also do explainer for client to client specs.
+* RG: I shared the current explainer with various communities and the client to client part was hard to explain.
+* HZ: since it is out of scope for the WG, should we just mention that eventually this will be requried?
+* HZ: I can reach out to the Solid team asking for the status.
+
+#### Clarifying "Achieve Them" in Deliverables Section
+URL: https://github.com/solid/solid-wg-charter/issues/45
+
+* VB: If no proposal to improve, can we close this issue or do you prefer to wait for a PR?
+* eP: More confortable if PAC handles this and everything WG charter related. He can ask us for help whenever needed.
+
+#### Does the value come from the protocol or the applications?
+URL: https://github.com/solid/solid-wg-charter/pull/71
+
+* eP: More practical to leave it to PAC
+
+### New Work Item C2C: Project Management
+URL: https://github.com/solid/specification/issues/641
+
+* eP: see also https://github.com/solid/specification/discussions/554
+* HZ: I was looking at contacts and it doesn't look very active. I'm afraid that we are starting work items which are going nowhere.
+* RG: I think the question is is there someone to work with Pavlik to work on it.
+* MLG: I see two levels of interop. The c2c ..., the other one is generic interop, where two apps from two different domains can interop.
+* eP: There is also overlap between the domains, we don't have hard boundries in real life and there is always interweaving.
+* HZ: is it Solid CG or contrib? I think CG is to general.
+* RG: what do you see the scope of CG? It seems protocol related work.
+* TT: it sounds like you ask for a task force within CG.
+* eP: if we already have sai and type indexes, I agree with having a new repo for that. I will create a PR to the shapes repo, we don't need a full spec. This will allow us to validate our assumptions. I will build on top of sai.
```

Comment:
```suggestion
* eP: if we already have SAI and type indexes, I agree with having a new repo for that. I will create a PR to the shapes repo; we don't need a full spec. This will allow us to validate our assumptions. I will build on top of SAI.
```